scout/commands/export/case.py

```diff
@@ -56,21 +56,20 @@ def cases(context, case_id, institute, reruns, finished, causatives, research_re
         case_obj = adapter.case(case_id=case_id)
         if case_obj:
             models.append(case_obj)
+        else:
+            LOG.info("No case with id {}".format(case_id))
 
     else:
         models = adapter.cases(collaborator=institute, reruns=reruns,
                            finished=finished, has_causatives=causatives,
                            research_requested=research_requested,
                            is_research=is_research, status=status)
-    if models.count():
-        LOG.info("No cases could be found")
+        if not models.count():
```

Comment:
```suggestion
        models = [case_obj for case_obj in models]
        if len(models) == 0:
```
Could you change to this so that it will work with PyMongo 3.7+ ?